plugins/modules/keycloak_user.py

```diff
@@ -0,0 +1,528 @@
+        type: bool
+    email_verified:
+        description:
+            - check the validity of user email.
+        default: false
+        type: bool
+        aliases:
+            - emailVerified
+    first_name:
+        description:
+            - The user's first name.
+        required: false
+        type: str
+        aliases:
+            - firstName
+    last_name:
+        description:
+            - User lastName.
+        required: false
+        type: str
+        aliases:
+            - lastName
+    email:
+        description:
+            - User email.
+        required: false
+        type: str
+    federation_link:
+        description:
+            - Federation Link.
+        required: false
+        type: str
+        aliases:
+            - federationLink
+    service_account_client_id:
+        description:
+            - Description of the client Application.
+        required: false
+        type: str
+        aliases:
+            - serviceAccountClientId
+    client_consents:
+        description:
+            - client Authenticator Type.
+        type: list
+        elements: dict
+        default: []
+        aliases:
+            - clientConsents
+        suboptions:
+            client_id:
+                description:
+                    - Client ID of the client role. Not the technical ID of the client.
+                type: str
+                required: true
+                aliases:
+                    - clientId
+            roles:
+                description:
+                    - List of client roles to assign to the user.
+                type: list
+                required: true
+                elements: str
+    groups:
+        description:
+            - List of groups for the user.
+        type: list
+        elements: dict
+        default: []
+        suboptions:
+            name:
+                description:
+                    - Name of the group.
+                type: str
+            state:
+                description:
+                    - Control whether the user must be member of this group or not.
+                choices: [ "present", "absent" ]
+                default: present
+                type: str
+    credentials:
+        description:
+            - User credentials.
+        default: []
+        type: list
+        elements: dict
+        suboptions:
+            type:
+                description:
+                    - Credential type.
+                type: str
+                required: true
+            value:
+                description:
+                    - Value of the credential.
+                type: str
+                required: true
+            temporary:
+                description:
+                    - If C(true), the users are required to reset their credentials at next login.
+                type: bool
+                default: false
+    required_actions:
+        description:
+            - requiredActions user Auth.
```

Comment:
I don't think this is possible to understand except if you already know what the option does.
```suggestion
            - RequiredActions user Auth.
```